src/OrbitGl/GlCanvas.cpp

```diff
@@ -1,16 +1,16 @@
+#include "GlCanvas.h"
```

Comment:
The licence declaration should be the first thing in a file:
https://google.github.io/styleguide/cppguide.html#File_Comments

Reply:
Also, it seems that you are including `GlCanvas` twice.

Reply:
Hm, seems that clang-format messed that up, thanks for catching.